pkgs/by-name/bu/butler/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  Cocoa,
+  ...
+}:
+
+# update instrctions:
+# - Check if butler version bug was fixed https://github.com/itchio/butler/issues/266
+# - if it's fixed, remove patch.
+# - if it was not fixed, follow steps below to regenerate the patch
+# - manually clone butler, change go.mod's version number to 1.18 at least
+# - run `go mod tidy` in the cloned repository.
+# - generate patch with `git diff > go.mod.patch`
+
+pkgs.buildGoModule rec {
+  pname = "butler";
+  version = "15.24.0";
+
+  buildInputs =
+    [
+      pkgs.brotli
+    ]
+    ++ lib.optionals pkgs.stdenv.isDarwin [
+      Cocoa
+    ];
+
+  patches = [ ./go.mod.patch ];
+
+  doCheck = false; # disabled because the tests don't work in a non-FHS compliant environment.
+
+  src = fetchFromGitHub {
+    owner = "itchio";
+    repo = "butler";
+    tag = "v${version}";
+    hash = "sha256-Gzf+8icPIXrNc8Vk8z0COPv/QA6GL6nSvQg13bAlfZM=";
+  };
+
+  vendorHash = "sha256-A6u7bKI7eoptkjBuXoQlLYHkEVtrl8aNnBb65k1bFno=";
+
+  meta = {
+    description = "Command-line itch.io helper.";
+    homepage = "http://itch.io";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.all;
```

Comment:
Not even Go works on all platforms. Leave it unset to default to all platforms Go supports

Reply:
Good to know, removed the definition.